### PR TITLE
fix: Fix querycoord panic during stop

### PR DESCRIPTION
**Pull request by weiliu1031**

### CODE REVIEW

**Comment by sunby** [comment]
/lgtm
sorry for this stupid mistake..

File: internal/querycoordv2/server.go [hunk 1]
```diff
@@ -447,7 +445,7 @@ func (s *Server) Stop() error {
 		s.session.Stop()
 	}
 
-	if s.session != nil {
+	if s.cluster != nil {
 		log.Info("stop cluster...")
 		s.cluster.Stop()
```
[the remark this answers is not in the document]
Comment (weiliu1031) [reply internal/querycoordv2/server.go:450]: cluster's `Stop` implement with 'sync.Once', it's idempotent

**Pull request merged**